DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/PromoteToAdminController.cs

```diff
@@ -47,9 +48,9 @@ ILogger<PromoteToAdminController> logger
         public IActionResult Index(int delegateId)
         {
             var centreId = User.GetCentreId();
-            var delegateUser = userDataService.GetDelegateUserById(delegateId);
+            var delegateUser = userDataService.GetDelegateUserCardById(delegateId);
 
-            if (delegateUser == null || delegateUser.CentreId != centreId)
+            if (delegateUser == null || delegateUser.CentreId != centreId || delegateUser.IsAdmin)

```

Comment:
As we also hide the button when the delegate does not have a password set can we add another check here so the users can't get around it with URL manipulation.